eng/testing/tests.mobile.targets

```diff
@@ -170,14 +174,18 @@
     </ItemGroup>
     <Error Condition="!Exists('$(MicrosoftNetCoreAppRuntimePackRidDir)')" Text="MicrosoftNetCoreAppRuntimePackRidDir=$(MicrosoftNetCoreAppRuntimePackRidDir) doesn't exist" />
     <WasmAppBuilder
-      AppDir="$(BundleDir)"
+      AppDir="$(AppDir)"
       InvariantGlobalization="$(InvariantGlobalization)"
       MicrosoftNetCoreAppRuntimePackDir="$(MicrosoftNetCoreAppRuntimePackRidDir)"
-      MainAssembly="$(PublishDir)WasmTestRunner.dll"
+      MainAssembly="$(PublishDir)$(MainAssembly)"
       MainJS="$(MonoProjectRoot)\wasm\runtime-test.js"
       Assemblies="@(Assemblies)"
       SatelliteAssemblies="@(WasmSatelliteAssemblies)"
       FilesToIncludeInFileSystem="@(WasmFilesToIncludeInFileSystem)" />
+    <WriteLinesToFile
```

Comment:
We should detect if we're running tests against v8 or the browser.  If we're running against the browser, then this task can be skipped.